packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -23,13 +23,13 @@ dependencies:
 
 dev_dependencies:
   auto_route_generator: ^1.0.2
-  build_runner: ^1.12.2
+  build_runner:
   # flutter_driver:
   #   sdk: flutter
   flutter_test:
     sdk: flutter
   mockito: ^5.0.3
-  rx_bloc_generator: ^3.0.2
+  rx_bloc_generator:
```

Comment:
Why we don't have a version here?

Reply:
The motivation of not having a version of rx_bloc_generator is that this is a template and supposably should be using the latest compatible version

Reply:
Then why `mockito` has a version (`^3.0.2`)?

Reply:
Done